$MirOS: ports/print/cups/patches/patch-berkeley_lprm_c,v 1.2 2006/06/21 19:25:33 bsiegert Exp $
--- berkeley/lprm.c.orig	Tue Sep  5 22:45:24 2006
+++ berkeley/lprm.c	Wed Nov  8 18:01:54 2006
@@ -243,7 +243,7 @@ main(int  argc,			/* I - Number of comma
       }
       else
       {
-        sprintf(uri, "ipp://localhost/jobs/%d", job_id);
+        snprintf(uri, sizeof(uri), "ipp://localhost/jobs/%d", job_id);
 	ippAddString(request, IPP_TAG_OPERATION, IPP_TAG_URI, "job-uri", NULL,
 	             uri);
       }
